style.css

```diff
@@ -0,0 +1,72 @@
+body {
+    padding: 0;
+    margin: 0;
+    background: #dae1eb url('images/background.jpg') no-repeat center center fixed;
```

Comment:
z4 in 2 zeilen:
background: #dae1eb;
background-image: url('images/background.jpg');